server/datastore/mysql/migrations/tables/20241030102721_AddAllLabelsToMDMProfileLabels.go

```diff
@@ -0,0 +1,41 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20241030102721, Down_20241030102721)
+}
+
+func Up_20241030102721(tx *sql.Tx) error {
+	// Add columns
+	_, err := tx.Exec(`ALTER TABLE mdm_configuration_profile_labels ADD COLUMN all_labels BOOL NOT NULL DEFAULT false`)
```

Comment:
I know I'm the one who suggested `all_labels`, but I'm thinking that maybe `require_all` would be a better column name. WDYT?